src/components/plots/barPlot.ts

```diff
@@ -38,6 +38,30 @@ export module Plots {
       this.attr("width", () => this._getBarPixelWidth());
     }
 
+    public x(): Plots.AccessorScaleBinding<X, number>;
+    public x(x: number | Accessor<number>): Bar<X, Y>;
+    public x(x: X | Accessor<X>, xScale: Scale<X, number>): Bar<X, Y>;
```

Comment:
The above I believe should already be declared on XYPlot and thus this should be unnecessary?